Makefile

```diff
@@ -0,0 +1,35 @@
+# Configure file directories
+BIN_DIR=./bin
+SRC_DIR=./src
+INCLUDE_DIR=./include
+EXEC=pathfinding.out
+
+CFLAGS += -Wall -g -I$(INCLUDE_DIR)/ -lm
+
+COMPILE=$(CC) $(CFLAGS) -c $^ -o $@
+LINK_EXE=$(CC) $(CFLAGS) -o $@ $^
```

Comment:
`$(CFLAGS)` should be added at the end. [Fix](https://github.com/CodePurble/A-Pathfinding-in-C-command-line/pull/1)

Reply:
That really does not make a difference and is purely a matter of style. The compiler treats them the same both ways.

Reply:
> That really does not make a difference and is purely a matter of style. The compiler treats them the same both ways.

No, it fails to build on my system

Reply:
It compiles fine for me (only a few warnings on deprecated library functions)
```sh
$ make
cc -Wall -g -I./include/ -lm -c src/file.c -o bin/file.o
cc -Wall -g -I./include/ -lm -c src/grid.c -o bin/grid.o
cc -Wall -g -I./include/ -lm -c src/node.c -o bin/node.o
cc -Wall -g -I./include/ -lm -c src/path.c -o bin/path.o
src/path.c: In function ‘main’:
src/path.c:45:4: warning: ‘ftime’ is deprecated [-Wdeprecated-declarations]
   45 |    ftime(&startTime);
      |    ^~~~~
In file included from src/path.c:5:
/usr/include/sys/timeb.h:39:12: note: declared here
   39 | extern int ftime (struct timeb *__timebuf)
      |            ^~~~~
src/path.c:95:6: warning: ‘ftime’ is deprecated [-Wdeprecated-declarations]
   95 |      ftime(&endTime);
      |      ^~~~~
In file included from src/path.c:5:
/usr/include/sys/timeb.h:39:12: note: declared here
   39 | extern int ftime (struct timeb *__timebuf)
      |            ^~~~~
cc -Wall -g -I./include/ -lm -o bin/pathfinding bin/file.o bin/grid.o bin/node.o bin/path.o

```

Here are my compiler details-
```
$ cc --version
cc (GCC) 10.1.0
```

Did it compile fine after moving `$(CFLAGS)` to the end??

Reply:
> Did it compile fine after moving `$(CFLAGS)` to the end??

Yes, after moving `$(CFLAGS)` to the end everything compiles fine:
![image](https://user-images.githubusercontent.com/51704055/85231701-2c09de80-b402-11ea-8473-5de5a5601434.png)

GCC version: `9.3.0`

Reply:
I've merged your PR. Everything should be fixed now.